examples/visual_studio/.buckconfig

```diff
@@ -11,7 +11,14 @@ fbsource = none
 buck = none
 
 [parser]
-target_platform_detector_spec = target:root//...->toolchains//:default
+target_platform_detector_spec = target:root//...->root//buck2_utils/platforms:windows_debug
```

Comment:
Why change this?

Reply:
Previously the visual_studio example was using a single target platform (toolchains//:default) that would change one of the constraint_values (debug/release) depending on a read_config call. In this PR I changed the example by creating 4 different target platforms (windows/linux and debug/release) in root//buck2_utils/platforms/BUCK. I only changed this because it seemed like the intended way to use Buck2. Now the target platform is specified by using the "--target-platforms" command line option.